#!/bin/sh -e
## wget-passive_ftp-default by Noèl Köthe <noel debian.org>
##
## DP: make passive-ftp the default

if [ $# -lt 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch} ${2:+-d $2}"

case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0

@DPATCH@
--- wget-1.9.1.orig/doc/sample.wgetrc
+++ wget-1.9.1/doc/sample.wgetrc
@@ -40,6 +40,7 @@
 # `passive' feature of FTP.  If you are behind such a firewall, you
 # can turn this on to make Wget use passive FTP by default.
 #passive_ftp = off
+passive_ftp = on

 # The "wait" command below makes Wget wait between every connection.
 # If, instead, you want Wget to wait only between retries of failed

